Packs/EDL/Integrations/EDL/README.md

```diff
@@ -99,6 +99,16 @@ Expected value is a string, supports newline characters (`\n`).
 
 When `PAN-OS: drop invalid URL entries` is enabled, any URL entry that is not compliant with PAN-OS URL format is dropped instead of rewritten.
 
+#### Exported Fields 
+For use with JSON and CSV formats - select specific Cortex XSOAR fields to export. If given the value 'all' - all Cortex XSOAR fields are exported. If empty - only value and type are exported.
+
+the optniol fields are:
+id - modified - sortValues - comments - indicator - value - source - sourceInstances - sourceBrands - investigationIDs - lastSeen - firstSeen - lastSeenEntryID - firstSeenEntryID - CustomFields - tags - expirationStatus - expirationSource - calculatedTime - lastReputationRun - modifiedTime - aggregatedReliability
+
```

Comment:
Let's make this a list of items (even tho it will be long)
```suggestion
Optional system fields are:
 - `id` 
 - `modified` 
 - `sortValues`
```